app-modules/authorization/tests/Feature/Filament/Resources/User/ViewUserTest.php

```diff
@@ -124,59 +124,6 @@
     expect(auth()->id())->toBe($second->id);
 });
 
-it('does not display the mfa_status Action for an external User', function () {
```

Comment:
@danharrin and @ankit-canyon do y'all think there is going to be any easy way to replicate these tests for this new way of doing it?

Reply:
You can at least assert the text with `assertSee()`